sources/datasources/clubs.js

```diff
@@ -40,31 +40,31 @@ class ClubAPI extends DataSource {
 			contactInfo: club.contactInfo
 		});
 
-		//Add nested types
-		const clubId = createdClub._id;
-		const accessArray = club.memberAccess;
-		const AccessLevels = new AccessLevelAPI();
-		if (accessArray != undefined && accessArray.length > 0) {
-			await Promise.all(
-				accessArray.map(async (accessItem, index) => {
-					accessItem.club=clubId;
-					await AccessLevels.addAccessLevel(accessItem);
-				})
-			);
-		}
+		// //Add nested types
+		// const clubId = createdClub._id;
+		// const accessArray = club.memberAccess;
+		// const AccessLevels = new AccessLevelAPI();
+		// if (accessArray != undefined && accessArray.length > 0) {
+		// 	await Promise.all(
+		// 		accessArray.map(async (accessItem, index) => {
+		// 			accessItem.club=clubId;
+		// 			await AccessLevels.addAccessLevel(accessItem);
+		// 		})
+		// 	);
+		// }
```

Comment:
Why comments all over this file

Reply:
that needs to be either moved or removed, stuck in a dilemma @roshankshaw can help :/